### PR TITLE
Updated deployment_get.go and routes.go

### PR DESCRIPTION
**Pull request by ashsenth**

### CODE REVIEW

File: go.mod
Comment (bobjac) [inline go.mod:1]: Looks like there are 2 versions of echo here

File: cmd/apiserver/handlers/deployment_get.go
Comment (bobjac) [inline cmd/apiserver/handlers/deployment_get.go:1]: Looks like a build issue with the parameter passed to mapper

**Review by bobjac — CHANGES_REQUESTED** [review]
Can you take a look at the build failure?

**Comment by kevinhillinger** [comment]
Fixing build failures to get to main and to our release

**Review by kevinhillinger — APPROVED** [review]

**Review by kevinhillinger — APPROVED** [review]

**Pull request merged**